nsxt/data_source_nsxt_policy_gateway_dns_forwarder.go

```diff
@@ -0,0 +1,40 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceNsxtPolicyGatewayDNSForwarder() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNsxtPolicyGatewayDNSForwarderRead,
+
+		Schema: map[string]*schema.Schema{
+			"id":           getDataSourceIDSchema(),
+			"display_name": getDataSourceDisplayNameSchema(),
+			"description":  getDataSourceDescriptionSchema(),
+			"path":         getPathSchema(),
+			"gateway_path": getPolicyPathSchema(false, false, "Gateway path"),
```

Comment:
Do we need to make gateway_path a required attribute?

Reply:
No, if the display name is unique, we can spare the user from specifying GW

Reply:
Sure, updated.

Reply:
OK, so this will not be enforced by us but up to the user. Thanks for the discussion!